gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeStatisticsPanel.java

```diff
@@ -62,26 +63,20 @@ public void onInitialize() {
 			}
 		}
 
-		add(new Label("graded", new StringResourceModel("label.statistics.gradedoutof",
-				null, new Object[] {
-						String.valueOf(allGrades.size()),
-						String.valueOf(gradeInfo.size()) }).getString()));
-		add(new Label("outof", String.valueOf(assignment.getPoints())));
+		add(new Label("graded", String.valueOf(allGrades.size())));
 
 		if (allGrades.size() > 0) {
 			Collections.sort(allGrades);
-			add(new Label("average", String.valueOf(calculateAverage(allGrades))));
-			add(new Label("median", String.valueOf(calculateMedian(allGrades))));
-			add(new Label("lowest", String.valueOf(Collections.min(allGrades))));
-			add(new Label("highest", String.valueOf(Collections.max(allGrades))));
-			add(new Label("variance", String.valueOf(calculateVariance(allGrades))));
-			add(new Label("deviation", String.valueOf(calculateStandardDeviation(allGrades))));
+			add(constructAverageLabel(allGrades, assignment));
```

Comment:
If this was:
`add(new Label("median", constructAverageLabel(allGrades, assignment)));`
The the method returned a string instead of the Label, that might be a bit more reusable. It's also a pattern we have elsewhere, sorry, could we do that small change?